src/lib/components/chat/MessageInput.svelte

```diff
@@ -1,7 +1,7 @@
 <script lang="ts">
 	import { toast } from 'svelte-sonner';
 	import { onMount, tick } from 'svelte';
-	import { settings } from '$lib/stores';
```

Comment:
I'd suggest not to use stores and to keep variables for dom elements in the components.

Reply:
what if the element is queried from outside the component?

Reply:
I see your point of possible performance gains but we should keep the code as-is.

Reply:
roger that